xbmc/guilib/GUITextLayout.cpp

```diff
@@ -315,7 +315,8 @@ std::wstring CGUITextLayout::BidiFlip(const std::wstring& text,
 
   // Convert to utf32, call bidi then convert the result back to utf16
   g_charsetConverter.wToUtf32(text, utf32logical);
-  g_charsetConverter.utf32logicalToVisualBiDi(utf32logical, utf32visual, false, false,
+  g_charsetConverter.utf32logicalToVisualBiDi(utf32logical, utf32visual, forceLTRReadingOrder,
+                                              false,
```

Comment:
I'm not really a specialist of this part, but seems makes sense, the `forceLTRReadingOrder` was unused before and on the `utf32logicalToVisualBiDi` call is this the third value.

So with this change the needed value becomes given to needed place :+1: